db/migrate/20200610083045_add_os_major_and_os_minor_to_hosts.rb

```diff
@@ -0,0 +1,8 @@
+class AddOsMajorAndOsMinorToHosts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :hosts, :os_major, :integer
+    add_column :hosts, :os_minor, :integer
```

Comment:
And here, if we just go ahead and name the columns `os_major_version`/`os_minor_version` we won't need the scoped_search renames.